conf/x86-64.src.default

```diff
@@ -0,0 +1,6 @@
+SOURCE_URL=https://packages.gitlab.com/gitlab/gitlab-ce/packages/debian/stretch/gitlab-ce___VERSION__-ce.0_amd64.deb/download.deb
+SOURCE_SUM=__SHA256_SUM__
+SOURCE_SUM_PRG=sha256sum
+SOURCE_FILENAME=gitlab-ce___VERSION__-ce.0_amd64.deb
```

Comment:
Can't we do something like  https://packages.gitlab.com/gitlab/gitlab-ce/packages/debian/stretch/gitlab-ce___VERSION__-ce.0___ARCH__.deb ans add an helper (which exists I think) to set the varable : arch. With that, we would have just one file, instead of two and two checksums as variable :-)

Reply:
I think it's complicated cause the ARM version is not in the subfolder .../gitlab/**gitlab-ce**/packages/**debian**/stretch/... but in .../gitlab/**raspberry-pi2**/packages/**raspbian**/stretch...
Moreover, the checksums are differents.

Reply:
yes... It's me :-) I didn't see the url in fact (was a fast review).

Reply:
No problem, thanks for the review :-)